.github/settings.yml

```diff
@@ -84,36 +84,35 @@ repository:
   # - name: org-admins
   #  permission: admin
 
-# branches:
-  # - name: main
+branches:
+   - name: main
     # https://developer.github.com/v3/repos/branches/#update-branch-protection
     # Branch Protection settings. Set to null to disable
-    # protection:
+     protection:
       # Required. Require at least one approving review on a pull request, before merging. Set to null to disable.
-      # required_pull_request_reviews:
+      required_pull_request_reviews:
         # The number of approvals required. (1-6)
-        # required_approving_review_count: 1
+        required_approving_review_count: 1
         # Dismiss approved reviews automatically when a new commit is pushed.
-        # dismiss_stale_reviews: true
+        dismiss_stale_reviews: true
         # Blocks merge until code owners have reviewed.
-        # require_code_owner_reviews: true
+        require_code_owner_reviews: true
         # Specify which users and teams can dismiss pull request reviews. Pass an empty dismissal_restrictions object to disable. User and team dismissal_restrictions are only available for organization-owned repositories. Omit this parameter for personal repositories.
         #dismissal_restrictions:
           # users: []
           # teams: []
       # Required. Require status checks to pass before merging. Set to null to disable
-      # required_status_checks:
+      required_status_checks:
         # Required. Require branches to be up to date before merging.
-        # strict: true
+        strict: true
         # Required. The list of status checks to require in order to merge into this branch
-        # contexts: []
+        contexts: []
       # Required. Enforce all configured restrictions for administrators. Set to true to enforce required status checks for repository administrators. Set to null to disable.
       # enforce_admins: true
       # Prevent merge commits from being pushed to matching branches
-      # required_linear_history: true
+      required_linear_history: true
       # Required. Restrict who can push to this branch. Team and user restrictions are only available for organization-owned repositories. Set to null to disable.
-      # restrictions:
-        # apps: []
-        # users: []
-        # teams:
-        #  - cisco-open/homebrew-tap-admins
```

Comment:
Just double-checking that the removal of the team here was intentional?

Reply:
There was a discussion between me and @detiber if this is needed. Reason I put this here was initially that I didn't want teams to self-approve their PRs but by doing that we would bottleneck them ... let me link the discussion

Reply:
https://github.com/cisco-open/homebrew-tap/pull/33#discussion_r1324551736

Reply:
Ty for the context!